deriva/core/ermrest_model.py

```diff
@@ -281,6 +281,97 @@ def ktup(k):
             provide_system
         )
 
+    @classmethod
+    def define_asset(cls, sname, tname, hatrac_template=None, column_defs=[], key_defs=[],
+                     fkey_defs=[], comment=None, acls={}, acl_bindings={}, annotations={},
+                     provide_system=True):
+        """Build an asset  table definition.
+
+          :param sname: the name of the schema for the asset table
+          :param tname: the name of the newly defined table
+          :param hatrac_template: template for the hatrac URL.  Will undergo substitution to template can include
+                 elmenents such at {{{MD5}}} or {{{Filename}}}
+          :param column_defs: a list of Column.define() results for extra or overridden column definitions
+          :param key_defs: a list of Key.define() results for extra or overridden key constraint definitions
+          :param fkey_defs: a list of ForeignKey.define() results for foreign key definitions
+          :param comment: a comment string for the table
+          :param acls: a dictionary of ACLs for specific access modes
+          :param acl_bindings: a dictionary of dynamic ACL bindings
+          :param annotations: a dictionary of annotations
+          :param provide_system: whether to inject standard system column definitions when missing from column_defs
+
+          These core asset table columns are generated automatically if
+          absent from the input column_defs.
+
+          - Filename: ermrest_curie, unique not null, default curie template "%s:{RID}" % curie_prefix
+          - URL: Location of the asset, unique not null.  Default template is:
+                    /hatrac/tname/{{{_URL.Filename}}}.{{{_URL.MD5}}} where tname is the name of the asset table.
+          - Length: Length of the asset.
+          - MD5: text
+          - Description: markdown, not null
+
+          However, caller-supplied definitions override the default.
+
+          In addition to creating the columns, this function also creates an asset annotation on the URL column to
+          facilitate use of the table by Chaise.
+          """
+
+        if not hatrac_template:
```

Comment:
I think should idiomatically be `if hatrac_template is None`.

Also, I think this default needs to be documented in the method docstring since the Python help summary will only show the default `hatrac_template=None`.